client/my-sites/checkout/src/components/item-variation-picker/jetpack-variant-dropdown-price.tsx

```diff
@@ -0,0 +1,90 @@
+import formatCurrency from '@automattic/format-currency';
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import { hasCheckoutVersion } from '@automattic/wpcom-checkout';
+import { useTranslate } from 'i18n-calypso';
+import { Discount, Label, Price, PriceTextContainer, Variant } from './styles';
+import type { WPCOMProductVariant } from './types';
+import type { FunctionComponent } from 'react';
+
+const isFreeMonthTrial = ( variant: WPCOMProductVariant ) =>
+	variant.termIntervalInMonths === 12 &&
+	variant.introductoryTerm === 'month' &&
+	variant.introductoryInterval === 1;
+
+const JetpackDiscountDisplay: FunctionComponent< {
+	isFreeMonthTrial?: boolean;
+	showIntroOffer?: boolean;
+	priceInteger: number;
+	currency: string;
+} > = ( { isFreeMonthTrial, showIntroOffer, priceInteger, currency } ) => {
+	const translate = useTranslate();
+
+	if ( isFreeMonthTrial ) {
```

Comment:
Here we should probably check to see if the trial is actually free rather than just a first month discount. We don't want the `One month free trial` to show up on a $1 first month discount sometime in the future